File: Lv1/두정수사이의합.py

def solution(a,b):
    answer=0
    for i in range(min(a,b),max(a,b)+1):
        answer+=i
    return answer
#for문을 돌릴 필요 없이 sum함수를 사용하는 것이 
print(solution(3,5))

File: solveWithJS/두정수사이의합.js
function solution(a, b) {
    let bigger = a>=b?a:b;
    let smaller = a<b?a:b;
    if(a==b) return a;
    let sum = 0;
    for(let i=smaller;i<=bigger;i++){
        sum+=i;
    }
    return sum;
}